src/main/daml/Daml/Finance/Interface/Types/Date/RollConvention.daml

```diff
@@ -30,4 +32,6 @@ data RollConventionEnum
   -- ^ Rolls on month end.
   | DOM Int
   -- ^ Rolls on the corresponding day of the month.
+  | NoRollConvention
```

Comment:
This is also not super-clear to me on what it means "not to have a roll convention"

Reply:
Exactly. Also in the case of `T` rolling does not apply. The schema has it as a mandatory element though: https://www.fpml.org/spec/fpml-5-11-3-lcwd-1/html/confirmation/schemaDocumentation/schemas/fpml-ird-5-11_xsd/complexTypes/CalculationPeriodDates/calculationPeriodFrequency.html
with a possible value of `NONE`.  With the Enum naming convention that would be None, which clashes in Daml.

Reply:
👍